platform/linux-generic/pktio/dpdk.c

```diff
@@ -163,6 +162,11 @@ static struct rte_mempool *mbuf_pool_create(const char *name,
 	unsigned num;
 	uint16_t data_room_size;
 
+	if (!(pool_entry->mem_from_huge_pages)) {
+		ODP_ERR("DPDK requires memory is allocated from huge pages\n");
+		return NULL;
```

Comment:
@Bill-Fischofer-Linaro If the memory is not from huge pages the physical address may not be available since the memory is not locked. DPDK NIC drivers require physical addresses to do DMA.

The tests have been passing because the packet pools are always allocated from huge pages. I noticed this problem when I happened to run out of free huge pages on my test system.